### PR TITLE
Assignment 2

### PR DESCRIPTION
**Pull request by Khanif3693**

### CODE REVIEW

File: Assignment_2/Index.html [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+        
+        $.ajax({
+        url: "https://pokeapi.co/api/v2/pokemon/ditto/",
+        type: 'GET',
+        dataType: 'json', // added data type
+        success: function(res) {
+            console.log(res);
+           var result=[];
+            for (x of res.game_indices)
```
Comment (AlthafPattan) [inline Assignment_2/Index.html]: Can you try `.map()` approach?

**Comment by AlthafPattan** [comment]
This is pretty good. 👍

File: Assignment_2/Index.html (hunk 1 above)
Comment (Khanif3693) [reply Assignment_2/Index.html]: Yes Sure

Comment (Khanif3693) [reply Assignment_2/Index.html]: Finished

Please let me know what do you feel?

**Comment by Khanif3693** [comment]
Hi Althaf,

Done with changes.
Let me know what do you think.

Thanks,
Hanif

File: Assignment_2/Index.html [hunk 2]
```diff
@@ -127,12 +127,16 @@
         dataType: 'json', // added data type
         success: function(res) {
             console.log(res);
-           var result=[];
-            for (x of res.game_indices)
-            {
-                result.push(x.version.name);
+           let result=res.game_indices.map(iter)
```
Comment (AlthafPattan) [inline Assignment_2/Index.html:130]: You can simplify like this `let result=res.game_indices.map((game) => game.version.name)`
So everything will be in one line. 
But this is also fine 👍

**Review by AlthafPattan — APPROVED** [review]
👍

File: Assignment_2/Index.html (hunk 2 above)
Comment (Khanif3693) [reply Assignment_2/Index.html:130]: Yes, It is working. I did it before but got confused.

**Pull request merged**